pages/wi/plan-your-trip.js

```diff
@@ -0,0 +1,201 @@
+import ContentSection from '../../components/shared/ContentSection';
+import ImageContainer from '../../components/shared/ImageContainer';
+import LinkButton from '../../components/shared/LinkButton';
+import TimelineSection from '../../components/HomePage/Timeline';
+
+const GET_EVENT = gql`
+  query getEvent($eventId: ID!) {
+    event(id: $eventId) {
+      id
+      name
+      slogan
+      startDate
+      endDate
+      venues {
+        id
+        name
+        address
+        city
+        state
+        zip
+      }
+      milestones {
+        title
+        description
+        dueDate
+      }
+      notifications {
+        id
+        shouldFeature
+        title
+        message
+        startDate
+        endDate
+        link
+        linkText
+      }
+    }
+  }
+`;
+
+const StyledImageContainer = styled(ImageContainer)`
+  padding: 2.5rem;
+  margin: 2rem;
+`;
+
+const Title = styled.h4`
+  margin-top: 0;
+  margin-bottom: 0.4rem;
+`;
+
+const ImageCell = styled(Cell)`
+  ${below.med`
+    text-align: center;
+  `};
+`;
+
+const HighlightImage = styled.img`
+  max-height: 40rem;
+  transform: scaleX(-1);
+  position: absolute;
+  top: 0;
+  margin-left: 6rem;
+
+  ${below.small`
+    margin-left: 0;
+    width: 90%;
+  `};
+
+  ${below.med`
+    position: relative;
+    margin-top: 2rem;
+  `};
+`;
+
+const contact = () => {
+  const { loading, error, data } = useQuery(GET_EVENT, {
+    variables: { eventId: 'ByE7Dc7eCGcRFzLhWhuI' },
+    onCompleted(d) {
+      return d;
+    },
+  });
+
+  if (loading) return null;
+  if (error) return null;
+
+  const { event } = data;
+  const formattedStartDate = moment(event.startDate).format(
+    'dddd, MMMM D, YYYY',
+  );
+  const formattedEndDate = moment(event.endDate).format('dddd, MMMM D, YYYY');
+
+  return (
+    <>
+      <Head>
+        <title key="title">Plan Your Trip - THAT Conference</title>
+      </Head>
+
+      <ContentSection>
+        <Grid columns={gridRepeat.xsmall}>
+          <Cell width={1}>
+            <h1 style={{ marginBottom: '0.5rem' }}>Plan Your Trip</h1>
+            <p className="medium-body-copy">
+              THAT Community spreads way beyond the midwest and we want to make
+              it easy for you to book and plan your travel to Summer Camp. Here
+              is the one stop show of dates, places and related travel goodness
+              to help get you to camp!
+            </p>
+          </Cell>
+          <ImageCell>
+            <HighlightImage src="/images/sasquatch_kayaking.png" />
+          </ImageCell>
+        </Grid>
+      </ContentSection>
+
+      <ContentSection>
+        <Grid columns={gridRepeat.xsmall} alignContent="center">
+          <StyledImageContainer>
+            <Title>When</Title>
+            <p style={{ flexGrow: '2' }}>
+              {formattedStartDate} -
+              <br />
+              {formattedEndDate}
+            </p>
+            <LinkButton href={DEFAULT_WIP_PAGE} label="Ticket Options" />
```

Comment:
DEFAULT_WIP_PAGE is 404'ing because it is going to `/wi/work-in-progress`

Reply:
Thanks for catching! Fixed until I can get the ticker page added.